cpp/src/decisiontree/batched-levelalgo/quantiles.cuh

```diff
@@ -35,8 +36,31 @@ namespace ML {
 namespace DT {
 
 template <typename T>
-__attribute__((visibility("hidden"))) __global__ void computeQuantilesKernel(
-  T* quantiles, int* n_bins, const T* sorted_data, const int max_n_bins, const int n_rows);
+__global__ void computeQuantilesKernel(
+  T* quantiles, int* n_bins, const T* sorted_data, const int max_n_bins, const int n_rows)
```

Comment:
Is this kernel used anywhere in the code base? If not, we should remove it

Reply:
It's used here in RandomForest through its host function wrapper https://github.com/rapidsai/cuml/blob/2f16814a7ef362e530886598167010f68d75d119/cpp/src/randomforest/randomforest.cuh#L149

Reply:
Oops, my bad. It's used by `computeQuantiles`.